apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -209,7 +212,9 @@ export default class ProjectManager {
       '/projects/' +
       this.lastChannel.projectType +
       '/' +
-      this.channelId
+      this.channelId +
+      '/' +
+      this.userId
```

Comment:
Why do we need the userId for the shareUrl here?

Reply:
Apparently it's not.  Removed!